c7n/tags.py

```diff
@@ -562,15 +566,32 @@ def process(self, resources):
 
         op = self.data.get('op', 'stop')
         tag = self.data.get('tag', DEFAULT_TAG)
-        date = self.data.get('days', 4)
+        days = self.data.get('days')
+        hours = self.data.get('hours')
+
+        n = datetime.now(tz=tzutc()).replace(minute=0)
+        if not days and hours:
```

Comment:
please put this logic in a method

Reply:
also this logic could be simplified to action_date string for day, and then add hour if it exists, instead of this four way clause expansion.

Reply:
i see how that would be easier... originally i thought that adding the %H:%M to the tag value would impact other unittests... but after simplifying and just going with it, it doesn't appear to have any impact at all.